tools/dds/dds-sniffer/rs-dds-sniffer.cpp

```diff
@@ -0,0 +1,474 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Did you do a `git mv`? Looks like everything changed...

Reply:
Yes, talked to you before renaming. dds-sniffer.cpp renamed rs-dds-sniffer.cpp, dds-sniffer.hpp renamed rs-dds-sniffer.h.

Reply:
Yes, used git mv on the file. Don't know why it is like that